class.jetpack-gutenberg.php

```diff
@@ -151,10 +158,6 @@ public static function enqueue_block_editor_assets() {
 	 * @return bool
 	 */
 	public static function should_load_blocks() {
-		if ( ! Jetpack::is_active() ) {
```

Comment:
This seems problematic though, since we now have blocks available even when Jetpack is not connected to WordPress.com.